src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,57 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 706px;
+  box-sizing: border-box;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    height: $day-size;
+    width: $day-size;
+
+    background-color: #eee;
+    border: 1px solid black;
+
+    transition-duration: 0.5s;
+
+    &::before {
+      content: counter(day);
+      counter-increment: day;
+    }
+
+    &:hover {
+      cursor: pointer;
+      transform: translateY(-20px);
+      background-color: #ffbfcb;
+    }
+  }
+
+  $days: (
+    mon: 0,
+    tue: 101px,
```

Comment:
The margins in the `$days` map should be multiples of `$day-size` to ensure consistency and avoid magic numbers like `101px`, `202px`, etc. You can calculate these margins based on the `$day-size` variable.